recipe/meta.yaml

```diff
@@ -26,15 +26,14 @@ requirements:
     - setuptools
   run:
     - python
-    - ipykernel >=4.1  # not a real dependency, but require the reference kernel
+    - traitlets
     - ipython_genutils
-    - jupyter_client >=4.1
     - jupyter_core
+    - jupyter_client >=4.1
     - pygments
-    - pyqt
-    - pyzmq >=17.1
+    - ipykernel >=4.1  # not a real dependency, but require the reference kernel
     - qtpy >=2.0.1
-    - traitlets !=5.2.1, !=5.2.2  # Skip traitlets versions that break Qtconsole on Windows
```

Comment:
The upstream source still points that we should skip some versions of `traitlets`, see https://github.com/jupyter/qtconsole/blob/7c739f4a9e7a1218c58a52374598b9d9202dd0c1/setup.py#L69

Reply:
Added! Thanks!